### ISSUE TITLE
Migrate away from JBox2D SNAPSHOT

### ISSUE THREAD

**Issue by freemo** [opennars/opennars#115]
Someone at some point decided to use the SNAPSHOT (unreleased) version of JBox2D. This means we cant release the project to maven central and any developers need to manual download, compile, and install it from the repo here: https://github.com/jbox2d/jbox2d

Obviously this needs to be fixed so we should migrate away fromt he SNAPSHOT version as soon as possible.

**Comment by patham9** [comment]
I agree, and would prefer to stick with stable releases of libraries, @automenta had a different philosophy here, I guess he wanted to always have the newest developed features at hand.
Currently jbox2D is used in nars_lab. Demos with physics will be great in the future although the current working examples in Launcher don't make use of it.

**Comment by freemo** [comment]
I think Seth just likes to try out ideas and experiment. He doesn't usually
like worrying about packing up a product. I can understand his enthisasm
but if we want to make a production. Ready product we will have to change
that pattern. This is by far the biggest place to start, all other
dependencies I found in maven with some minor compromises.
